seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -5393,6 +5393,54 @@
         }
       }
     },
+    {
+      "id": "xf2222c1x11fedafarb1blll",
```

Comment:
This id looks a bit off, do we not have a standard way of generating them for new challenges? @QuincyLarson @Hkuz if you guys can confirm?

Reply:
@raisedadead Yes - use this code instead. I generated it using the `ObjectId()` command in the MongoDB shell:

`5a2efd662fb457916e1fe604`